src/Bridge/Symfony/Bundle/Action/SwaggerUiAction.php

```diff
@@ -95,6 +100,13 @@ private function getContext(Request $request, Documentation $documentation): arr
             'spec' => $this->normalizer->normalize($documentation, 'json', ['base_url' => $request->getBaseUrl()]),
         ];
 
+        if (!$this->oauthRedirectUrl) {
+            $this->oauthRedirectUrl = $request->getScheme() . '://' .
+                $request->getHttpHost() .
+                $request->getBasePath() .
+                $this->assetPackages->getUrl('bundles/apiplatform/swagger-ui/oauth2-redirect.html');
```

Comment:
I suggest to generate the URL directly in the Twig template using the helpers if not defined in the config

Reply:
Do you mean in your index template `src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig` ? 

I'm not sure if this is such a good idea, because anyone who overwrites the template without extending it has to take care of a default value, or am i missing something?

If you mean i should create a new twig template for the swagger-ui's helper (`oauth2-redirect.html`), that would mean i would have also to create a controller/route for it and we also would no longer simply copy the original file and would have to transfer any further changes made by the swagger team manually into the template. Not sure if there will be any changes soon, just a thought of me.

Reply:
> Do you mean in your index template `src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig` ?

Yes. I would just include `{{ asset('bundles/apiplatform/swagger-ui/oauth2-redirect.html' }}`.
It's cleaner (no need to create a dependency to the Assets component in the controller), and not a big deal to copy/paste if needed.

Reply:
If an absolute URL is mandatory: `{{ absolute_url(asset('bundles/apiplatform/swagger-ui/oauth2-redirect.html')) }}`